ProcessLib/ProcessVariable.h

```diff
@@ -81,8 +81,9 @@ class ProcessVariable
 		for (auto& config : _neumann_bc_configs)
 		{
 			config->initialize(searcher);
-			bcs++ = new NeumannBc<GlobalSetup>(*config,
-			                                   std::forward<Args>(args)...);
+			bcs++ = std::unique_ptr<NeumannBc<GlobalSetup>>{
```

Comment:
This one is explicitly for template argument lookup. Without GlobalSetup function argument I'd need to specify all template arguments when calling, which is cumbersome.